tests/test_mcp_wrapper.py

```
@@ -0,0 +1,151 @@
+                capture_output=True,
+                text=True,
+                timeout=timeout,
+            )
+            return proc.stdout, proc.stderr, proc.returncode
+        finally:
+            os.unlink(child_path)
+
+    def test_valid_json_passes_through_stdout(self):
+        """Valid JSON lines should appear on stdout."""
+        child = 'import json, sys; print(json.dumps({"jsonrpc":"2.0","id":1})); sys.stdout.flush()'
+        stdout, stderr, _ = self._run_wrapper(child)
```

Ruff (0.15.12)

[warning] Unpacked variable `stderr` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)